### ISSUE TITLE
Package doesn't support soft wrap.

### ISSUE THREAD

**Issue by xanfus** [camel-chased/editor-background#49]
Text shadows project upon background as if no soft wrap is present.

**Comment by Fusseldieb** [comment]
Yeah, folding is broken... The shadow does not "recalculate" when something folds...

**Comment by TiriaAndersen** [comment]
Is there a way to fix this?

**Comment by Fusseldieb** [comment]
> Is there a way to fix this?

Literally years ago I spent **hours** looking for a fix (inside the code), even pushed some changes inside this repo if I'm not mistaken, but it ultimatively didn't solve anything.

Try Visual Studio Code, it has everything Atom has and more. Personal opinion, of course.

**Comment by TiriaAndersen** [comment]
@Fusseldieb Gotcha. Thanks for the heads up XD

**Comment by xanfus** [comment]
> Is there a way to fix this?

That would has to do with data, which isn't in file, but in memory, in intrinsic rendering applications.
